cmd/scheduler/app/server.go

```diff
@@ -63,6 +64,13 @@ func Run(opt *options.ServerOption) error {
 		return err
 	}
 
+	if opt.PluginsDir != "" {
+		err := framework.LoadCustomPlugins(opt.PluginsDir)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
err will be catched by `main` function in `cmd/scheduler/app/main.go`

```go

	if err := app.Run(s); err != nil {
		fmt.Fprintf(os.Stderr, "%v\n", err)
		os.Exit(1)
	}
```

Reply:
> err will be catched by `main` function in `cmd/scheduler/app/main.go`
> 
> ```go
> 	if err := app.Run(s); err != nil {
> 		fmt.Fprintf(os.Stderr, "%v\n", err)
> 		os.Exit(1)
> 	}
> ```

yeah, i mean add some custom info such as "get plugin path error, err msg: %v\n" instead of output raw os msg. It' more friendly when view log.

Reply:
OK